package.json

```diff
@@ -27,7 +27,7 @@
     ]
   },
   "dependencies": {
-    "@catena-x/portal-shared-components": "^3.0.4",
+    "@catena-x/portal-shared-components": "^3.0.6",
```

Comment:
Please change to version 3.0.7, latest one available

Reply:
@evegufy Made the changes